### PR TITLE
READY: Use a config file for styling nodes, links etc. instead of magic numbers and colors everywhere.

### PR DESCRIPTION
**Pull request by rkeulemans**
Solves #150

### CODE REVIEW

File: TriblerGUI/widgets/trustpage/js/main.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ var state = {
     request_pending: false,
     x: width / 2,
     y: height / 2,
-    focus_pk: "self",
+    focus_pk: "10",
```
Comment (TimSpeelman) [inline TriblerGUI/widgets/trustpage/js/main.js]: Too much magic for me. Shouldnt this be self?

File: TriblerGUI/widgets/trustpage/js/style_config.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            y : 24
+        },
+        circle : {
+            radius : 20,
+            cx : 0,
+            cy : 0,
+            cursor : "pointer"
+        }
+    }
+};
```
Comment (TimSpeelman) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: ...

File: TriblerGUI/widgets/trustpage/js/main.js (hunk 1 above)
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/main.js]: As Tim said, please remove this debugging purpose.

File: TriblerGUI/widgets/trustpage/js/style_config.js (hunk 2 above)
Comment (MaxVanDeursen) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: newline

File: TriblerGUI/widgets/trustpage/js/main.js (hunk 1 above)
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/main.js]: Why is "10" here

File: TriblerGUI/widgets/trustpage/js/drawing.js [hunk 3]
```diff
@@ -137,16 +141,27 @@ function drawLinks(svg, data) {
  * @returns the width of the link
  */
 function getStrokeWidth(link, data) {
-    // Map relative to the minimum and maximum total transition links of the graph
-    var difference = data.max_transmission - data.min_transmission;
-    if(difference === 0) {
-        return 6;
-    }
+
```
Comment (vandenheuvel) [inline TriblerGUI/widgets/trustpage/js/drawing.js]: You can remove this newline.

**Pull request merged**